README.md

```diff
@@ -105,9 +108,110 @@ The *RRbot* URDF files can be found in the `description` folder of `ros2_control
          joint1/position [unclaimed]
          joint2/position [unclaimed]
    state interfaces
-          joint1/position
-          joint2/position
+         joint1/position
+         joint2/position
+   ```
+
+3. Check which controller are running using:
+   ```
+   ros2 control list_controllers
+   ```
+   You should get something like:
    ```
+   forward_position_controller[forward_command_controller/ForwardCommandController] unconfigured
+   joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+   ```
+
+4. Check [Controlles and moving hardware](#Controlles-and-moving-hardware) section to move *RRBot*.
+
+
+### Example 1: "Industrial Robots with only one interface"
+
+- Launch file: rrbot_system_position_only.launch.py
+- Command interfaces:
+  - joint1/position
+  - joint2/position
+- State interfaces:
+  - joint1/position
+  - joint2/position
+
+Available controllers:
+  - `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
+  - `forward_position_controller[forward_command_controller/ForwardCommandController]` (position)
+
+Available launch-file options:
+  - `use_fake_hardware:=true` - start `FakeSystem` instead of hardware.
+    This is a simple simulation that mimics joint command to their states.
+    This is useful to test *ros2_control* integration and controllers without physical hardware.
+
+
+### Example 1: "Industrial Robots with only one interface" (Gazebo simulation)
+
+- **TBA**
+
+
+### Example 2: "Robots with multiple interfaces"
+
+- Launch file: rrbot_system_position_only.launch.py
+- Command interfaces:
+  - joint1/position
+  - joint2/position
+  - joint1/velocity
+  - joint2/velocity
+  - joint1/acceleration
+  - joint2/acceleration
+- State interfaces:
+  - joint1/position
+  - joint2/position
+  - joint1/velocity
+  - joint2/velocity
+  - joint1/acceleration
+  - joint2/acceleration
+
+Available controllers:
+  - `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
+  - `forward_position_controller[position_controllers/JointGroupPositionController]`
+  - `forward_velocity_controller[velocity_controllers/JointGroupVelocityController]`
+  - `forward_acceleration_controller[forward_command_controller/ForwardCommandController]`
+  - `forward_illegal1_controller[forward_command_controller/ForwardCommandController]`
+  - `forward_illegal2_controller[forward_command_controller/ForwardCommandController]`
+
+Notes:
+  - The example shows how to implement multi-interface robot hardware taking care about interfaces used.
+    The two illegal controllers demonstrate how hardware interface declines faulty claims to access joint command interfaces.
+
+
+### Example 10: "Differential drive mobile robot"
```

Comment:
For this demo the default rviz configuration file did not set up correctly the fixed frame (I'm guessing it's `odom`) nor did it showed the Robot Model.

After fixing this and sending the velocity command the demo runs fine.

One thing I noticed is the velocity being correctly updated in the `/joint_states` while the wheel positions is fixed to zero. 

```
header:
  stamp:
    sec: 1623246367
    nanosec: 753984445
  frame_id: ''
name:
- right_wheel_joint
- left_wheel_joint
position:
- 0.0
- 0.0
velocity:
- 50.0
- 43.33333333333333
effort:
- .nan
- .nan
```

Reply:
Thanks @jordan-palacios I will try to propose a fix by tomorrow morning.